Beeper/BarcelonaMautrixIPC/Runtime/BLStandardIO.swift

```diff
@@ -65,6 +65,14 @@ public extension FileHandle {
     }
 }
 
+extension FileHandle: MautrixIPCInputChannel {
+    public func listen(_ cb: @escaping (Data) -> ()) {
+        self.handleDataAsynchronously(cb)
+    }
+}
+
+extension FileHandle: MautrixIPCOutputChannel {}
```

Comment:
Could we get a comment here explaining how `FileHandle` actually conforms to this protocol? Is it a method implemented elsewhere within barcelona? Is it a method built-in to `FileHandle`? Just feels a bit confusing to me

Reply:
I agree, I was planning to refactor the whole implementation soonish so I didn't bother, but might as well add it since it's a small thing.